test/unit/dsl/objects_test.rb

```diff
@@ -396,17 +423,20 @@
       control.title = "Variable Control Important Title"
       control.descriptions[:default] = "The most variable control the world has ever seen"
       control.impact = 1.0
+      # TODO: doubled up on subject... probably skippable w/ some logic
       control.to_ruby.must_equal '
 control "variable.control.id" do
   title "Variable Control Important Title"
   desc  "The most variable control the world has ever seen"
   impact 1.0
-  a = command("which grep")
-  describe a do
-    its("exit_status") { should eq 0 }
+  let(:a) { command("which grep") }
```

Comment:
I like the use of `let`.

The way `subject` is used makes controls look quite different from the ideal examples you see on the front page of inspec.io.
I think customers might question why we have this `subject { a }` instead of just `describe a do`

Reply:
Again, this is only for generated code.